howtos/otel/how_to_integrate_opentelemetry_with_pubsubplus_api.py

```diff
@@ -0,0 +1,150 @@
+from solace.messaging.resources.topic_subscription import TopicSubscription
+from solace.messaging.resources.queue import Queue
+from solace.messaging.receiver.message_receiver import MessageHandler
+from solace.messaging.resources.topic import Topic
+from solace.messaging.receiver.inbound_message import InboundMessage
+from solace.messaging.messaging_service import MessagingService
+from solace.messaging.receiver.persistent_message_receiver import PersistentMessageReceiver
+
+from opentelemetry import propagate
+from opentelemetry import context
+from opentelemetry import trace, baggage
+from opentelemetry.semconv.trace import SpanAttributes, MessagingDestinationKindValues
+from opentelemetry.trace import StatusCode, Status, SpanKind
+from typing import List
+
+from solace_otel.messaging.trace.propagation import InboundMessageCarrier, InboundMessageGetter, OutboundMessageCarrier, OutboundMessageSetter
+
+TRACER_NAME = "my_tracer"
```

Comment:
can I propose to change the tracer name to `myAppTracer` to follow the same convention we have for the go API sampler

Reply:
https://github.com/SolaceSamples/solace-samples-go/tree/main/patterns/otel-tracing